zk.el

```diff
@@ -828,27 +798,25 @@ Optionally call a custom function by setting the variable
 (defun zk--links-in-note-list ()
   "Return list of zk files that are linked from the current buffer."
   (let* ((zk-alist (zk--alist))
-         (zk-ids (zk--id-list))
+         (zk-ids (zk--id-list nil zk-alist))
```

Comment:
the zk-alist arg won't be used if the str arg is nil

Reply:
That's what d6be688ea0280576875a262e2883282d30779fe5 in the other pull request is for.